shared/bundle_analysis/report.py

```diff
@@ -92,7 +95,7 @@ class BundleAnalysisReport:
     def __init__(self, db_path: Optional[str] = None):
         self.db_path = db_path
         if self.db_path is None:
-            _, self.db_path = tempfile.mkstemp()
+            _, self.db_path = tempfile.mkstemp(prefix="bundle_analysis_")
```

Comment:
make the temp file created for BA more identifiable. Allows API to clean of the files easier.